contents/handbook/small-teams/replay/index.mdx

```diff
@@ -16,22 +16,27 @@ We build the [heads-up display](https://en.wikipedia.org/wiki/Head-up_display) w
 
 ## Q4 2023 Goals
 
-### 🎯 Goal 1 : Project "NoteForce Exploration 3000" (Ben, David)
+### 🎯 Goal 1: 10 Happy Android Replay Users 
 
-Why? Pairing up with Michael, Thomas, and Cory from Team Product Analytics to focus on not just releasing Notebooks, but integrating them seamlessly with PostHog 3000 to make it a great experience to explore any and all data across PostHog.
+We'll also release an alpha of iOS Mobile Replay.
 
-### 🎯 Goal 2 : Mobile Session Replay (Manoel, Paul)
+Why? 
 
-Why? - The people want it and we have the right skills in house to do a good job of this. Replay is the main focus, but making mobile a first-class citizen of PostHog is the real goal.
+We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
 
-Follow along in [our Q4 MegaIssue](https://github.com/PostHog/posthog/issues/17803)
+### 🎯 Goal 2: Reduce churn
+
+Why? - Churn is higher than we'd like, this is a signal we're not getting some things right. We're not serving our users as well as we can until we're getting this right.
+
+### 🎯 Goal 3: First-class mobile support in app.posthog.com
+
+Why? We're making the mobile SDKs equivalent to the web SDKs. We want to make sure that the mobile SDKs are first-class citizens in app.posthog.com so that it is easy to use and explore your mobile data.
 
 ## Areas of responsibility
 
 - **Session Replay** - recording browser or mobile sessions to be replayed later
 - **Toolbar** - a floating helper on your web app that helps to toggle feature flags, display heatmaps and provide contextual analysis
-- **Notebooks** - a way to explore your data in a free format environment, for example to add comments to a session replay
```

Comment:
Should we leave notebooks within Replay until we decide which team should own it? Right now we don't have a Product OS team and realistically most bugs / improvements are going to come my way

Reply:
Not really outside of the Product Analytics you can do inside a notebook.

Where we got to on this was not to do any active development on the notebooks platform but to be around for bugs & support if other teams want to implement / improve their products experience in notebooks